lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.t

```diff
@@ -0,0 +1,81 @@
+use strict;
+use warnings;
+
+BEGIN {
+    $ENV{NO_LSF}                         = 1;
+    $ENV{UR_DBI_NO_COMMIT}               = 1;
+    $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+}
+
+use above 'Genome';
+use Genome::SoftwareResult;
+
+use Test::More;
+use Test::Exception;
+use File::Compare qw(compare);
+use File::Basename qw(basename);
+use Genome::Utility::Test qw(compare_ok);
+use Genome::File::Vcf::Differ;
+use Genome::Test::Factory::SoftwareResult::User;
+use Genome::Test::Data qw(get_test_file);
+
+
+	my $test_dir = __FILE__.".d";
+	
+	my $output = Genome::Sys->create_temp_directory();
+        
+	my $reference_fasta = get_test_file('NA12878', 'human_g1k_v37_20_42220611-42542245.fasta');
+	my $bam = get_test_file('NA12878', 'NA12878.20slice.30X.aligned.bam');
+	my $bam2 = get_test_file('NA12878', 'NA12878.20slice.30X.aligned.bam');
+	my $split_bam = get_test_file('NA12878','NA12878.20slice.30X.splitters.bam');
+	my $discordant_bam = get_test_file('NA12878','NA12878.20slice.30X.discordants.bam');
+
+ 
+	my $pkg2 = 'Genome::Model::Tools::DetectVariants2::SpeedseqSv';
+	
+	my $refbuild_id = 101947881;
+	my $result_users = Genome::Test::Factory::SoftwareResult::User->setup_user_hash(reference_sequence_build_id => $refbuild_id,);
+
+	my $params = "-R:$reference_fasta,-g,-d,-o:Hello";
+
+	print "Here is the base name of the bam variable: \t";
+	print basename($bam);
+	print "\n\n";
```

Comment:
The best way to print debug information in tests is with `diag`.  This is guaranteed to print the messages in a way that won't break the test output.